README.md

```diff
@@ -67,28 +67,7 @@ This library requires Apache Spark 2.0+ and Amazon Redshift 1.0.963+.
 
 For version that works with Spark 1.x, please check for the [1.x branch](https://github.com/databricks/spark-redshift/tree/branch-1.x).
 
-You may use this library in your applications with the following dependency information:
-
-**Scala 2.10**
-
-```
-groupId: com.databricks
-artifactId: spark-redshift_2.10
-version: 3.0.0-preview1
-```
-
-**Scala 2.11**
-```
-groupId: com.databricks
-artifactId: spark-redshift_2.11
-version: 3.0.0-preview1
```

Comment:
Should we leave this (updated with 4.0.0) ?

Reply:
does 4.0.0 works ? I was under the impression that the last spark-redshift was 3.0.0-preview1.

From what I understand the last release of spark-redshift community is master-SNAPSHOT only.

Thus, that is the reason why I removed it and merged this section with what was under it (snapshot).

Reply:
Oh, we have 4.0.0:
https://mvnrepository.com/artifact/io.github.spark-redshift-community/spark-redshift

I meant maybe we can update that to be:
groupId: io.github.spark-redshift-community
artifactId: spark-redshift_2.11
version: 4.0.0

Reply:
Oh I didn't know 4.0.0 was there, I'll update the readme then.

Reply:
So I only see 2.11 for release 4.0.0 but snapshot has 2.10 and 2.11, is that accurate ?

Reply:
Where did you see that ? 
I think SNAPSHOT has only 2.11 too:
https://oss.sonatype.org/#nexus-search;quick~spark-redshift-community

I think adding the following should be enough?
```
groupId: io.github.spark-redshift-community
artifactId: spark-redshift_2.11
version: 4.0.0
```

Reply:
The README contains group ids that point to https://mvnrepository.com/artifact/com.github.databricks/spark-redshift
(see https://github.com/spark-redshift-community/spark-redshift/blob/master/README.md#snapshot-builds)

So that's why I was asking because I was able to use "com.github.databricks" %% "spark-redshift" % "master-SNAPSHOT" inside my sbt.

So ok, I will assume theses are valid:
```
groupId: io.github.spark-redshift-community
artifactId: spark-redshift_2.11
version: 4.0.0
```
```
groupId: io.github.spark-redshift-community
artifactId: spark-redshift_2.11
version: 4.0.0-SNAPSHOT
```